pygfx/controllers/_panzoom.py

```diff
@@ -45,6 +59,26 @@ def __init__(
         # Save initial state
         self.save_state()
 
+    @property
+    def mode(self) -> str:
+        if self.maintain_aspect:
+            return "maintain-aspect"
+        else:
+            return "freescale"
```

Comment:
This can just be a bool. Something like:
```suggestion
    @property
    def maintain_aspect(self) -> bool:
        return self._maintain_aspect
```